components/02-elements/swatch/_swatch.scss

```diff
@@ -0,0 +1,86 @@
+$swatch-font-size        : $font-size-small !default;
+$swatch-margin           : 15px !default;
+$swatch-width            : 48px !default;
+$swatch-transition       : $transition-base !default;
+
+$swatch-background       : $white !default;
+$swatch-color            : $gray-light !default;
```

Comment:
Isn't it a label color?

Reply:
label? It's just color of text inside the swatcher

Reply:
So it's the label of the swatch.

Reply:
ok, variable changed to `$swatch-label-color`